app/src/main/java/gitlin/kothub/github/api/dsl/User.kt

```diff
@@ -23,6 +23,17 @@ class User(override val level: Int): ActorFields {
         )
     }
 
+    fun starredRepositories(first: Int? = null, body: UserConnection.() -> Unit) {
+
+        val connection = UserConnection(nextLevel())
```

Comment:
starredRepository creates a repository connection - check the Github GraphQL documentation.
https://developer.github.com/v4/reference/object/user/